job_templates/snippet/system_groovy_script_block.xml.em

```diff
@@ -0,0 +1,15 @@
+<hudson.plugins.groovy.SystemGroovy plugin="groovy@@2.2">
+@[if command]@
+      <source class="hudson.plugins.groovy.StringSystemScriptSource">
+        <script plugin="script-security@@1.76">
+          <script>@ESCAPE(command)</script>
+          <sandbox>false</sandbox>
+        </script>
+      </source>
+@[end if]@
+@[if script_file]@
+      <source class="hudson.plugins.groovy.FileSystemScriptSource">
+        <scriptFile>@ESCAPE(script_file)</scriptFile>
+      </source>
+@[end if]@
```

Comment:
Was this snippet copied from ros_buildfarm? Is there an anticipated use case for this branch of the snippet?